src/main.js

```diff
@@ -60,13 +58,14 @@ function changeDisplays() {
   formSection.classList.toggle('hidden');
   currentPageTitle.innerText = 'Current Activity';
   updateTimerPage();
+  formSection.reset();
+
```

Comment:
I think Sue and I are looking at a different solution for this and we can add that during the merge conflict.